platforms/software/dependency-management/src/test/groovy/org/gradle/api/internal/catalog/LibrariesSourceGeneratorTest.groovy

```diff
@@ -393,7 +393,7 @@ ${nameClash { noIntro().kind('dependency bundles').inConflict('one.cool', 'oneCo
             TestUtil.objectFactory(),
             Stub(Supplier)) {
             @Override
-            protected Problems getProblemService() {
+            protected InternalProblems getProblemService() {
```

Comment:
💅 I think calling it `getProblemsService` would be better - `getProblemService` sounds more like "there is a problem with this service" than "this is a service for handling problems" to me.

Also indentation.